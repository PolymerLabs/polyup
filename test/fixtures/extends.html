<link rel="import" href="../polymer/polymer.html">

<polymer-element name='extends-builtin' extends='input' noscript>
</polymer-element>

<polymer-element name='extends-custom' extends='paper-input' noscript>
</polymer-element>
