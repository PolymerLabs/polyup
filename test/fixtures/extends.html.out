<link rel="import" href="../polymer/polymer.html">

<dom-module id="extends-builtin">
</dom-module>
<script>
  Polymer({
    is: 'extends-builtin',
    extends: 'input'
  });
</script>

<!--
    TODO(polyup): Inheriting from other custom elements is not yet supported.
    See: https://www.polymer-project.org/1.0/docs/migration.html#inheritance
 -->
<dom-module id="extends-custom">
</dom-module>
<script>
  Polymer({
    is: 'extends-custom',
    extends: 'paper-input'
  });
</script>
